static_src/test/unit/stores/user_store.spec.js

```diff
@@ -134,22 +135,25 @@ describe('UserStore', function () {
       expectedUsers = [
         {
           guid: userGuidA,
-          roles: { [spaceGuid]: ['space_developer'] }
+          roles: { [spaceGuid]: ['space_developer'] },
+          space_roles: [ 'space_developer' ]
```

Comment:
is there a reason we need this new attribute `space_roles` when we already have a mapping between the space guid and the roles that person has for the guid?